src/components/common/NumericEditor.vue

```diff
@@ -18,10 +19,13 @@
 </template>
 
 <script>
-import { mapMutations } from 'vuex'
-
 export default {
   props: {
+    state: {
```

Comment:
To make it official: Is this to be the pattern moving forward? All editors will then receive state variable that matches up to bootstrap's API?

I'd like to suggest we rename to validationState for clarity, since the word state is pretty overloaded already!

Reply:
@bzabos , yes, we will use that pattern for @poojahpatil90 's ticket to match up to bootstrap's API.
I will rename the prop to `validationState`